operations/src/test/java/edu/unc/lib/boxc/operations/impl/order/OrderValidatorFactoryTest.java

```diff
@@ -58,4 +58,10 @@ public void forSetOrderRequestTest() {
         var request = OrderRequestFactory.createRequest(OrderOperationType.SET, PARENT_UUID, Arrays.asList(CHILD1_UUID));
         assertTrue(factory.createValidator(request) instanceof SetOrderValidator);
     }
+
+    @Test
+    public void forClearOrderRequestTest() {
+        var request = OrderRequestFactory.createRequest(OrderOperationType.CLEAR, PARENT_UUID, Arrays.asList(CHILD1_UUID));
```

Comment:
can leave off the last parameter, for ClearOrder requests we only need to say what object we are clearing the order of.
```suggestion
        var request = OrderRequestFactory.createRequest(OrderOperationType.CLEAR, PARENT_UUID);
```